I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 278, in script
    python:
ScriptError: could not find label 'script_ep5'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "D:\michelle\renpy-8.0.3-sdk\renpy\bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "D:\michelle\renpy-8.0.3-sdk\renpy\main.py", line 660, in main
    run(restart)
  File "D:\michelle\renpy-8.0.3-sdk\renpy\main.py", line 146, in run
    renpy.execution.run_context(True)
  File "D:\michelle\renpy-8.0.3-sdk\renpy\execution.py", line 938, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 278, in script
    python:
  File "D:\michelle\renpy-8.0.3-sdk\renpy\script.py", line 927, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'script_ep5'.

Windows-10-10.0.22000 AMD64
Ren'Py 8.0.3.22090809
Shoot Tomorrow 1.0
Sat Sep  2 11:09:08 2023
